CnvStrUtils.pas

```diff
@@ -48,9 +48,10 @@ function AddStrings(const Items, Items2 : array of string): String; overload;
 function IndexOf(const Items : array of string; const Item : String;
     CaseSensitive : boolean = false): Integer;
 procedure DeleteFromArray(var Items: TStringArray; AElementIndex: integer);
-function ExtractValue(const s : string): string;
-function ExtractName(const s : string): string;
-function SplitNameAndValue(const s : string; var AName, AValue : string): Boolean;
+function ExtractValue(const s : string; const AEq : string = '='): string;
+function ExtractName(const s : string; const AEq : string = '='): string;
+function SplitNameAndValue(const s : string; var AName, AValue : string;
```

Comment:
@fgomez, I don't recommend making this change in isolation.
I think this function is exposed to PAX and DelphiScript. If import files are not updated, pax and delphiscripts that use this function will become broken.
Either you update all import files affected (which could be even PAX import files for Delphi 2007 and DelphiXE8 or 4) or you create an overloaded function.
You will also have to be careful with the overloaded function approach if you use the same function name just changing the parameter list, because you have to be sure that the PAX and delphiscrip registrations take the address of the prior function. I think if you declare an overloaded function after the first function might be enough, but you should not do this change without first testing PAX and Delphiscript behavior, if exposed to any of the two.

In the case of PAX, there's specific tests in PowerView for CnvStrUtils.pas unit